### PR TITLE
Update script to handle multiple gentxs

### PR DESCRIPTION
**Pull request by JakeHartnell**

### CODE REVIEW

File: scripts/validate-gentx.sh [hunk 1]
```diff
@@ -44,7 +44,7 @@ if [ $LEN_GENTX -eq 0 ]; then
 else
     set -e
 
-    echo "GentxFile::::"
+    echo "GentxFiles::::"
     echo $GENTX_FILE
```
Comment (wlouie1) [inline scripts/validate-gentx.sh:48]: References to GENTX_FILE can be removed

Comment (spacepotahto) [inline scripts/validate-gentx.sh:48]: all references to GENTX_FILE can be removed

**Review by nullmames — APPROVED** [review]
fingers crossed

**Review by spacepotahto — APPROVED** [review]
Generally looks good, there are some unused vars here and there, like "GENTX_FILE" and "amountquery" and the commented out logic, but those can be cleaned up later

**Pull request merged**